### DISCUSSION TITLE
Exporting Date Ranges

### DISCUSSION THREAD

**Discussion by dwhitnall** [hargata/lubelog#689] [Q&A]
I've had a look about the demo instance and was trying to see if there is a way to export records for a date range.
As I would like to utilise this as a logbook for tax reasons, but will require to get totals for the fiscal year.
Is there a way to do this or would this be something on the cards to happen.

**Comment by hargata** [comment]
See #761 

This feature will be included in the next release.

**Comment by hargata** [comment]
Released
